inline/LinkTrait.php

```diff
@@ -248,6 +248,9 @@ protected function renderImage($block)
 
 	protected function identifyReference($line)
 	{
+		if ('' == $line) {
```

Comment:
this should never happen, as empty lines are skipped by the Parser. See https://github.com/cebe/markdown/commit/7e6fce6b656603c1289a45a41bca065ee8cf3058 which seems to fix the issue you have reported.